src/tests/utils/convert.js

```diff
@@ -140,6 +140,44 @@ describe('convert render functions', () => {
     });
   });
 
+  describe('render step', () => {
+    it('should fill with trigger default description', () => {
```

Comment:
I believe you're missing a few `done()` calls here, since the tests are async.

Reply:
By [returning the Promise](https://github.com/zapier/zapier-platform-cli/pull/170/files#diff-3b1a6bd6d07ddfd7a243e89ee5d1ce92R151), Mocha will handle the asynchronous code. I confirmed it by messing around the assertion and it did break the tests.

Reply:
Oh, I wasn't aware of that! Thanks! Feel free to merge, then.